app/lib/frontend/templates/misc.dart

```diff
@@ -181,97 +180,102 @@ String renderMiniList(List<PackageView> packages) {
 }
 
 /// Renders the tags using the pkg/tags template.
-String renderTags({
-  @required PackageView package,
-  @required SearchQuery searchQuery,
-}) {
+String renderTags({@required PackageView package}) {
   final tags = package.tags;
   final sdkTags = tags.where((s) => s.startsWith('sdk:')).toSet().toList();
-  final List<Map> tagValues = <Map>[];
+  final tagValues = <Map>[];
   final tagBadges = <Map>[];
-  if (package.isExternal) {
-    // no tags added
-  } else if (package.isAwaiting) {
+  if (package.isDiscontinued) {
     tagValues.add({
-      'status': 'missing',
-      'text': '[awaiting]',
+      'status': 'discontinued',
+      'text': 'discontinued',
       'has_href': false,
-      'title': 'Analysis should be ready soon.',
+      'title': 'Package was discontinued.',
     });
-  } else if (package.isDiscontinued) {
+  }
+  if (package.tags.contains(PackageTags.isUnlisted)) {
     tagValues.add({
-      'status': 'discontinued',
-      'text': '[discontinued]',
+      'status': 'unlisted',
+      'text': 'unlisted',
       'has_href': false,
-      'title': 'Package was discontinued.',
+      'title': 'Package was unlisted.',
```

Comment:
Let's make this more elaborate, something like:
> Package is unlisted, this means that while the package is still publicly available the author has decided that it
> should not appear in search results with default search filters. This is typically done because this package is meant
> to support another package, rather than being consumed directly.